packages/components/multiselect/src/Multiselect.test.tsx

```diff
@@ -277,3 +277,49 @@ describe('Multiselect with select all', () => {
     expect(mockOnSelectAll).toHaveBeenCalled();
   });
 });
+
+describe('Multiselect with clear selection', () => {
+  it('renders the button only when there is more than one element selected', async () => {
+    const onClear = jest.fn();
+
+    const produce: Fruit[] = [
+      { id: 1, name: 'Apple 🍎', isChecked: false, isDisabled: false },
+      { id: 2, name: 'Ananas 🍍', isChecked: false, isDisabled: false },
+      { id: 3, name: 'Avocado 🥑', isChecked: true, isDisabled: false },
+    ];
+    renderComponent(
+      { onClearSelection: onClear, currentSelection: ['Avocado 🥑'] },
+      produce,
+      true,
+    );
+
+    expect(
+      within(screen.getByTestId('cf-multiselect-current-selection')).getByText(
+        'Avocado 🥑',
+      ),
+    ).toBeInTheDocument();
+    expect(screen.queryByLabelText('Clear selection')).not.toBeInTheDocument();
+  });
```

Comment:
suggestion: can you add a test for when there's nothing selected the button does not get rendered?